tester/tester.go

```diff
@@ -39,9 +39,11 @@ type Contract struct {
 
 	ExpectedHTTPCode     int               `json:"http_code_is" yaml:"http_code_is"`
 	ExpectedResponseBody string            `json:"response_body_contains" yaml:"response_body_contains"`
+	ExpectedResponse     []string 		   `json:"response_contains" yaml:"response_contains"`
```

Comment:
perhaps the plural `ExpectedResponses` would be better since this is a slice

Reply:
Done